en/docs/guides/identity-federation/apple.md

```diff
@@ -0,0 +1,158 @@
+
+## Configuring the identity provider
+To configure the WSO2 Identity Server as an identity provider that uses Apple for federated authentication:
+
+1. On the Management Console, go to **Identity > Identity Providers > Add**.
+
+    The **Add New Identity Provider** screen appears.
+
+    ![Add New Identity Provider screen]({{base_path}}/assets/img/guides/add-new-identity-provider-screen.jpeg)
+
+2. Enter a meaningful name as the **Identity Provider Name**, e.g., `AppleSignIn`.
+
+3. Expand **Federated Authenticators** > **OAuth2/OpenID Connect Configuration** and enter the required values as given below.
+
+    ![OAuth2/OpenID Connect Federated Authenticator Configuration screen]({{base_path}}/assets/img/guides/oauth2-oidc-federated-authenticator-configuration-screen.jpeg)
+
+    <table>
+        <thead>
+            <tr>
+                <th>Field</th>
+                <th>Description</th>
+                <th>Sample Value</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>Enable OAuth2/OpenIDConnect</td>
+                <td>Selecting this option enables OAuth2/OpenID Connect to be used as an authenticator for users provisioned to the WSO2 Identity Server.</td>
+                <td>Selected</td>
+            </tr>
+            <tr>
+                <td>Default</td>
+                <td>Selecting this option signifies that OAuth2/OpenID Connect is used as the main/default form of authentication. Selecting this removes the selection made for any other **Default** checkboxes for other authenticators.</td>
+                <td>Selected</td>
+            </tr>
+            <tr>
+                <td>Client Id</td>
+                <td>This is the <code>client key</code> of your Apple application.</td>
+                <td><code>idp.demologin.com</code></td>
+            </tr>
+            <tr>
+                <td>Client Secret</td>
+                <td>This is the <code>client secret</code> of your Apple application.</td>
+                <td></td>
+            </tr>
+            <tr>
+                <td>Authorization Endpoint URL</td>
+                <td>This is the authorization endpoint URL of Apple Sign In.</td>
+                <td><code>https://appleid.apple.com/auth/authorize</code></td>
+            </tr>
+            <tr>
+                <td>Token Endpoint URL</td>
+                <td>This is the token endpoint URL of Apple Sign In.</td>
+                <td><code>https://appleid.apple.com/auth/token</code></td>
+            </tr>
+            <tr>
+                <td>Callback URL</td>
+                <td>This is the service provider's URL to which authorization codes are sent. Upon successful authentication, the browser should be redirected to this URL. The URL should be specified in the following format: <code>https://:/acs</code></td>
```

Comment:
Shall we change this to match with other federated IDPs?
https://is.docs.wso2.com/en/latest/guides/identity-federation/facebook/#:~:text=This%20is%20the%20URL%20to%20which%20the%20browser%20should%20be%20redirected%20after%20the%20authentication%20is%20successful.%20This%20should%20be%20the%20commonauth%20endpoint%20of%20Identity%20server.

Reply:
ack will update this